pkgs/development/python-modules/stem/default.nix

```diff
@@ -23,6 +23,10 @@ buildPythonPackage rec {
     hash = "sha256-Oc73Jx31SLzuhT9Iym5HHszKfflKZ+3aky5flXudvmI=";
   };
 
+  patches = [
+    ./assertRaisesRegexp-to-assertRaisesRegex.patch
```

Comment:
maybe use `SubstituteInPlace` instead of doing a giant patch?

Reply:
I think patches is better as it will not fail silently. I've also made a PR to upstream https://github.com/torproject/stem/pull/155